LICENSE

```diff
@@ -1,27 +1,7 @@
-Copyright (c) 2015, First Issues
-All rights reserved.
+Copyright 2015 First Issues
```

Comment:
Can you add `MIT License` at the top. Similar to this [LICENSE](https://github.com/arshadkazmi42/currency-symbols/blob/master/LICENSE)